runtime/runtime/src/tests/apply.rs

```diff
@@ -2793,3 +2837,169 @@ fn test_deploy_and_call_local_receipts() {
         ActionErrorKind::FunctionCallError(FunctionCallError::MethodResolveError(_))
     );
 }
+
+#[test]
+fn test_transaction_ordering_with_apply() {
+    let alice_key1 = InMemorySigner::test_signer(&alice_account());
+    let bob_signer = InMemorySigner::test_signer(&bob_account());
+    let alice_key2 = InMemorySigner::from_seed(alice_account(), KeyType::ED25519, "seed");
+
+    let tx1 = SignedTransaction::send_money(
```

Comment:
consider renaming these to refer to the signer so that it is easier for the reader to follow.  E.g. this could be `alice_tx1`.

Reply:
based on the comment below, this tx should be called `alice_invalid_tx` or something similar.

Reply:
done